cli/src/test/kotlin/io/spine/protodata/cli/app/MainSpec.kt

```diff
@@ -103,12 +126,17 @@ class MainSpec {
         val project = ProjectProto.getDescriptor()
         val testProto = TestProto.getDescriptor()
         val request = codeGeneratorRequest {
-            protoFile.addAll(listOf(
-                project.toProto(),
-                testProto.toProto(),
-                TestOptionsProto.getDescriptor().toProto(),
-                OptionsProto.getDescriptor().toProto()
-            ))
+            protoFile.addAll(
+                listOf(
+                    project.toProto(),
+                    testProto.toProto(),
+                    TestOptionsProto.getDescriptor().toProto(),
+                    AstProto.getDescriptor().toProto(),
+                    FileProto.getDescriptor().toProto(),
+                ) +
+                spineOptionProtos() +
```

Comment:
This fragment is formatted weirdly. The [same piece in `LoggingLevelSpec`](https://github.com/SpineEventEngine/ProtoData/pull/309/files#diff-f448e9b8864c0dcd30ed0e3ec592ded83a7435e33aec10982366d9eb209d0bebR109) looks better to me.